### PR TITLE
Feature/xaptc 57

### PR DESCRIPTION
**Pull request by frankquinn**
changed response types and added delete test file methods

### CODE REVIEW

File: uiServices/src/services.go [hunk 1]
```diff
@@ -28,7 +28,7 @@ func validateFileNameAndHeader(rw http.ResponseWriter, req *http.Request, header
 
 	if len(name) <= 1 {
 		logrus.Error("File Not Found")
```
Comment (jenny-flynn) [inline uiServices/src/services.go]: Why is a single character incorrect for a file name? is A.xml an invalid filename?

File: uiServices/src/services_test.go [hunk 2]
```diff
@@ -120,6 +120,9 @@ func TestValidJsonPost(t *testing.T) {
 	reader := strings.NewReader(validJson)
 	r.HandleFunc("/configs", postConfigs)
 
+	//remove file if exists
+	os.Remove(os.Getenv("GOPATH") + "/src/github.com/xtracdev/automated-perf-test/uiServices/test/ServiceTestConfig.xml")
```
Comment (jenny-flynn) [inline uiServices/src/services_test.go]: maybe consider mocking out the file creation and use the godog tests to test actual creation of the file. not essential for this PR but worth investigating.

**Comment by frankquinn** [comment]
that <= 1 was a typo, should have been < 1 . I changed it now

**Comment by jenny-flynn** [comment]
Can we add a unit test to test that filename of 1 character is valid. Thanks!

File: uiServices/src/services.go [hunk 3]
```diff
@@ -26,9 +26,9 @@ func getConfigHeader(req *http.Request) string {
 
 func validateFileNameAndHeader(rw http.ResponseWriter, req *http.Request, header, name string) bool {
 
-	if len(name) <= 1 {
+	if len(name) < 1 {
 		logrus.Error("File Not Found")
```
Comment (jenny-flynn) [inline uiServices/src/services.go]: rename this log to something like "file name is empty"

**Pull request merged**